### PR TITLE
Add <PackingListInfo> element for <Product> entity

### PR DESCRIPTION
**Pull request by thijsw**
Implemented the PackingListInfo element from the API:

screen shot 2017-03-09 at 12 59 11

### CODE REVIEW

File: src/Entity/PackingListInfo.php [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+    /**
+     * @param string $number
+     * @param string $amount
+     *
+     * @return $this
+     */
+    public function addPackageAssociated($number, $amount)
+    {
+        $object = new \stdClass();
```
Comment (stefandoorn) [inline src/Entity/PackingListInfo.php]: Can you create a PackageAssociated entity and have that as an argument for this method? That entity can have a toNode method also again, so we keep the flow of letting the entity create it's node itself.

**Comment by stefandoorn** [comment]
@thijsw Could you have a look into this PR? :-)

**Comment by stefandoorn** [comment]
Ping @thijsw

**Comment by gabrielbull** [comment]
@thijsw This library has no maintainer and is looking for one.

**Pull request closed without merging**